### ISSUE TITLE
Suggest for syntax docs

### ISSUE THREAD

**Issue by suitupstan** [rajlang/Syntax#1]
I think we should maintain a documentation on the site rather than a code block 😄

**Comment by Rajaniraiyn** [comment]
Yeah @krishguptadev. This is just the demo. The actual language draft will be ready in the future. This code block will change anytime only for language reference.
